04_setup_ironic.sh

```diff
@@ -93,12 +99,15 @@ fi
 # Create pod
 sudo podman pod create -n ironic-pod 
 
-# Start Ironic and inspector
-sudo podman run -d --net host --privileged --name ironic \
-    --pod ironic-pod \
-    -v $IRONIC_DATA_DIR/dnsmasq.conf:/etc/dnsmasq.conf \
-    -v $IRONIC_DATA_DIR/html/images:/var/www/html/images \
-    -v $IRONIC_DATA_DIR/html/dualboot.ipxe:/var/www/html/dualboot.ipxe \
-    -v $IRONIC_DATA_DIR/html/inspector.ipxe:/var/www/html/inspector.ipxe ${IRONIC_IMAGE}
+# Start dnsmasq, http, and dnsmasq containers using same image
+sudo podman run -d --net host --privileged --name dnsmasq  --pod ironic-pod \
+     -v $IRONIC_DATA_DIR:/shared --entrypoint /bin/rundnsmasq ${IRONIC_IMAGE} 
+
+sudo podman run -d --net host --privileged --name httpd --pod ironic-pod \
+     -v $IRONIC_DATA_DIR:/shared --entrypoint /bin/runhttpd ${IRONIC_IMAGE} 
+
+sudo podman run -d --net host --privileged --name ironic --pod ironic-pod \
+     -v $IRONIC_DATA_DIR:/shared ${IRONIC_IMAGE} 
```

Comment:
@bfournie Hey did you look at using a newer podman version then launching the pod and containers via `podman play`?

Not a blocker but I was hoping we could move towards that, could be cleaner and more well aligned with the pod definition we'll need on the deployed cluster?

Not blocking, just wondered if you'd had a chance to try it, because last time I tried it didn't work as expected..

Reply:
I haven't tried using a new podman/libpod, it would be useful to also have the podman generate command which we currently don't have to create a consistent yaml definition with k8s on deployed cluster.  Agreed not a blocker for this patch but I'm creating a card to upgrade podman and use the new commands.